udata_front/theme/gouvfr/assets/js/components/AdminPublishButton/AdminPublishButton.vue

```diff
@@ -1,13 +1,28 @@
 <template>
-  <a :href="dataset_publishing_form_url" class="fr-btn fr-icon-add-line fr-btn--icon-left">
+  <a :href="link" class="fr-btn fr-icon-add-line fr-btn--icon-left">
     {{ t("Publish on") }}
     <img class="fr-ml-1v" :src="logo" loading="lazy" :alt="title" />
   </a>
 </template>
 <script setup lang="ts">
 import { useI18n } from 'vue-i18n';
 import logo from "../../../../templates/svg/logotype-white.svg";
-import { dataset_publishing_form_url, title } from '../../config';
+import { dataset_publishing_form_url, reuse_publishing_form_url, title } from '../../config';
+
+const props = defineProps<{
+  type: "dataset" | "reuse"
+}>();
 
 const { t } = useI18n();
+
+const link = getLink();
+
+function getLink() {
+  switch(props.type) {
+    case 'dataset':
+      return dataset_publishing_form_url;
+    case 'reuse':
+      return reuse_publishing_form_url
+  }
+}
```

Comment:
Need to be a `computed` (even if there is no reason for the `type` prop to change after mounting the component :-D )